drivers/hmis/app/controllers/ac_hmis/reports_controller.rb

```diff
@@ -0,0 +1,25 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module AcHmis
+  class ReportsController < Hmis::BaseController
```

Comment:
Let's move this over to the controller thats currently used for prevention assessment